internal/pkg/docker/dockerfile/lex.go

```diff
@@ -0,0 +1,241 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockerfile
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+	"unicode"
+)
+
+// instructionName identifies the name of the instruction.
+type instructionName int
+
+const (
+	instrErr         instructionName = iota // an error occurred while scanning.
+	instrHealthCheck                        // a HEALTHCHECK instruction.
+	instrExpose                             // an EXPOSE instruction.
+	instrEOF                                // done scanning.
+)
+
+const (
+	exposeInstrMarker     = "expose "      // start of an EXPOSE instruction.
```

Comment:
or `instrMarkerExpose` and `instrMarkerHealthcheck`?

Reply:
renamed to `markerExposeInstr` and `markerHealthCheckInstr`!